multi-agent/elements/llms/openai/validator.py

```
@@ -98,19 +117,109 @@ def validate(
+        """
+        try:
+            available_models = client.models.list()
+            return {m.id for m in available_models.data}
+        except (NotFoundError, APIStatusError) as e:
+            # 404, 501, or other errors indicate /v1/models not supported
+            status = getattr(e, 'status_code', None)
+            logger.error(f"Models endpoint not available (HTTP {status}): {e}")
+            if status in (404, 501, None):
+                return None
+            raise
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)